lms/djangoapps/student_dashboard/tests.py

```diff
@@ -0,0 +1,36 @@
+import ddt
+import mock
+import factory
+
+from django.db.models import signals
+from django.test import TestCase, RequestFactory
+
+from common.lib.nodebb_client.categories import ForumCategory
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+from lms.djangoapps.student_dashboard.views import get_joined_communities, get_recommended_xmodule_courses, \
+    get_enrolled_past_courses
+
+
+@ddt.ddt
+class TestStudenDashboard(TestCase):
```

Comment:
I guess we have decided to use `pytests` for all future unit tests of PhilU, @zahid-arbisoft please correct me if I am wrong>

Reply:
@ahmed-faraz46 it will work with pytests as well.

Reply:
@zahid-arbisoft it is using `SharedModuleStoreTestCase`